README.markdown

```diff
@@ -24,3 +24,51 @@ Quero ajudar!
 
 [![Bitdeli Badge](https://d2weczhvl823v0.cloudfront.net/rennerocha/dojopuzzles/trend.png)](https://bitdeli.com/free "Bitdeli Badge")
 
+Rodando o projeto!
+----------------------
+
+### Requeriments
+
+   - Python 3.7
```

Comment:
**issue**: We don't have a pinned version of Python yet, but all my development process I am using Python 3.8, so this is the expected Python version. When #30 is finished this will be more clear.